index.html

```diff
@@ -13,20 +13,20 @@
   </head>
   <body>
     <header>
-      <section id="session">
-        <a class="sess-link" href="#"><i class="sess-icon"><img class="face" src="images/icons/facebook-icon.png" alt="facebook icon"></i></a>
-        <a class="sess-link" href="#"><i class="sess-icon"><img class="twit" src="images/icons/twitter-icon.png" alt="twitter icon"></i></a>
-        <a class="sess-link" href="#"><p class="sess-icon">English</p></a>
-        <a class="sess-link" href="#"><p class="sess-icon">MyPage</p></a>
-        <a class="sess-link" href="#"><p class="sess-icon">Logout</p></a>
-      </section>
+      <ul id="session">
+        <li href="#"><a class="sess-link"><i><img class="face" src="./images/icons/facebook-icon.png" alt="facebook icon"></i></a></li>
+        <li href="#"><a class="sess-link"><i><img class="twit" src="./images/icons/twitter-icon.png" alt="twitter icon"></i></a></li>
+        <li href="#"><a class="sess-link">English</a></li>
+        <li href="#"><a class="sess-link">MyPage</a></li>
+        <li href="#"><a class="sess-link">Logout</a></li>
+      </ul>
       <label class="menu-btn">
-        <input type="checkbox">
+        <input id="checkbox" type="checkbox">
         <span class="burguer"></span>
         <nav>
-          <a href="#"><img class="nav-logo" src="images/content/ts-logo.png" alt="Tecate Supremo logo"></a>
+          <a href="#"><img class="nav-logo" src="./images/content/ts-logo.png" alt="Tecate Supremo logo"></a>
           <ul>
-            <li class="nav-item"><a href="pages/about.html" class="nav-link">About</a></li>
+            <li class="nav-item"><a href="./pages/about.html" class="nav-link">About</a></li>
             <li class="nav-item"><a href="#" class="nav-link">Program</a></li>
```

Comment:
- [ OPTIONAL] Well done on the Capstone Project! A small suggestion, consider adding the white border to the hover effect as per the original design. This will enhance the overall professionalism of your project. :wink:  :wink: 

Your desing :
![Screenshot from 2023-02-10 23-25-06](https://user-images.githubusercontent.com/88894525/218200357-0b26862f-43a9-48c4-926b-9cabe2a5a12e.png)


figma Design:
![figmaHover](https://user-images.githubusercontent.com/88894525/217973052-fe490603-d507-4ecc-a14c-e09550119489.png)